### PR TITLE
HPCC-32096 Expose MP listen queue somaxconn for containerized envs

### PR DESCRIPTION
**Pull request by mckellyln**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [x] Existing deployed queries will not be broken
  - [x] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [x] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by mckellyln** [comment]
Probably should do the same for mpStart/mpEnd port range.

File: system/mp/mpcomm.cpp [hunk 1]
```diff
@@ -2165,6 +2165,11 @@ CMPConnectThread::CMPConnectThread(CMPServer *_parent, unsigned port, bool _list
     parent->mpTraceLevel = getComponentConfigSP()->getPropInt("logging/@detail", InfoMsgThreshold);
     if (listen)
     {
+        if (getComponentConfigSP()->hasProp("expert/@mpSoMaxConn"))
+            mpSoMaxConn = getComponentConfigSP()->getPropInt("expert/@mpSoMaxConn", 0);
```
Comment (jakesmith) [inline system/mp/mpcomm.cpp:2169]: trivial: personally I'd remove the , 0 - it's the default anyway, but hasProp has established there's a value.. so slightly misleading.

**Review by jakesmith — APPROVED** [review]
@mckellyln - looks good, this is a simple/safe change, so could go into 9.4 if there's a need

File: system/mp/mpcomm.cpp (hunk 1 above)
Comment (mckellyln) [reply system/mp/mpcomm.cpp]: ok, I've removed the 0 arg in an amend as its such a simple change.

**Comment by mckellyln** [comment]
ok to merge.
LMK if we should re-target base branch to candidate-9.4.x

**Comment by ghalliday** [comment]
Yes, please rebase to 9.4.x

**Comment by mckellyln** [comment]
rebased to candidate-9.4.x

**Pull request merged**